### DISCUSSION TITLE
Intercept logs before view

### DISCUSSION THREAD

**Discussion by AmraniCh** [opcodesio/log-viewer#296] [Q&A]
Hey, thanks for the package!
I want to know if it possible to intercept logs before they passed to the view ? I want to filter the logs based on some criteria related to my app business logic, is there any thing I can do to achieve that ?

**Comment by arukompas** [comment]
@AmraniCh Hey!

Hm, that's an interesting idea! I feel like an access gate would work here, but at the same time you probably also want the log counts to reflect that filter too. 

At the moment that's not possible, but I'll note this down, seems like an interesting feature to have. :) 

Thanks !
